examples/webxr_vr_sandbox.html

```diff
@@ -218,6 +235,10 @@
 				torus.rotation.y = time * 5;
 
 				renderer.render( scene, camera );
+				stats.update();
+
+				// Canvas elements doesn't trigger DOM updates, so we have to update the texture
+				statsMesh.material.map.update();
```

Comment:
The canvas element resizes?

Reply:
ah, could be clearer in the description, canvas can trigger dom changes, this line is needed because render updates on the 2d canvas context needs a manual update call.

Reply:
Hmm, is this for stats.js or in general?

Reply:
we will need to call .update() only if the input dom element is an actively updating canvas.

for all other dom types, MutationObserver calls the `.update()` automatically. (I was thinking maybe add a html marquee element as another HTMLMesh example)